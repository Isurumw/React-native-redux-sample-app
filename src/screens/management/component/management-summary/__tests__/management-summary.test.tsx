import React from 'react';
import {render} from 'react-native-testing-library';

import ManagementSummary from '../management-summary';

describe('render the management summary card', () => {
  it('the summary card component renders correctly', () => {
    const component = render(
      <ManagementSummary
        icon={require('assets/images/key.png')}
        title="test title"
        subTitle="test sub title"
      />,
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
