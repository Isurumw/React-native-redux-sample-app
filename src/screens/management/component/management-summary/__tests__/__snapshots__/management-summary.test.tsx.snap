// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the management summary card the summary card component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#EBEEF2",
        "borderWidth": 1,
        "flexDirection": "row",
      },
      undefined,
    ]
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/key.png",
      }
    }
    style={
      Object {
        "height": 90,
        "resizeMode": "contain",
        "width": 80,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 1,
        "marginHorizontal": 8,
        "marginVertical": 16,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#2E3C57",
          "fontFamily": "Inter-Bold",
          "fontSize": 17,
          "letterSpacing": -0.22,
          "lineHeight": 22,
          "marginBottom": 4,
        }
      }
    >
      test title
    </Text>
    <Text
      style={
        Object {
          "color": "#2E3C57",
          "fontFamily": "Inter-Regular",
          "fontSize": 15,
          "letterSpacing": -0.13,
        }
      }
    >
      test sub title
    </Text>
  </View>
</View>
`;
